### DISCUSSION TITLE
Integration with Penpot

### DISCUSSION THREAD

**Discussion by wolfred24** [ParthJadhav/Tkinter-Designer#310] [Ideas]
Could this tool can be integrated with PENPOT the new open source alternative to figma?

**Comment by zbuhrer** [comment]
This would be a game changer for me, yes please 🙏

**Comment by RobertLSnead** [comment]
agree, penpot would be far better, with figma you are limited to only one project for the free version, I am surprised no one has built a python gui designer in python yet, maybe it's not possible?
